handbook/engineering/leadership/index.md

```diff
@@ -69,6 +69,18 @@ Our long term vision is to build an organization that attracts and retains the b
 - Owner: TBD.
 - Status: Not started. Needs more discussion.
 
+## Communication
+
+We have a 1hr sync with product and engineering every Tuesday at 9am PT ([meeting notes](https://docs.google.com/document/d/10fEh3Cw0ENKjFYDP-4OoLZbOZTVAtMpifc2WM_1mGVU/edit)).
```

Comment:
Rather than specify the time of the meeting, how about maintaining just the document link.  On a personal note, I was hoping to move this meeting - and wouldn't want to maintain the handbook as a result of times shifting.

Reply:
I think it is valuable to document the time here (whatever it may be) because it can be helpful for future teammates (e.g. future web team manager) to know. It is easy to update the docs here if we update the meeting day/time on a recurring basis. This doesn't prevent us from rescheduling ad hoc on any given week.